style.css

```diff
@@ -0,0 +1,186 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-size: 16px;
+    background-color: #FE9D8D;
+    font-family: "Press Start 2P", sans-serif;
+}
+
+h1 {
+    color: #373C99;
+}
+
+.landingTopContainer {
+    display: flex;
+    justify-content: space-between;
+}
+
+.landingLeft {
+    width: 20%;
+}
+
+.landingTitleContainer {
+    justify-content: space-between;
+}
+
+.leaderSurfboardContainer {
+    display: flex;
+}
+
+.mainContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+#mainGameContainer {
+    margin: 5%;
+    width: 1000px;
+    height: 600px;
+    position: relative;
+    border-radius: 10px;
+    background: linear-gradient(#f9855d,#fdd288,#ffffff);
+    overflow: hidden;
+}
+
+.waves {
+    position: relative;
+    width: 100%;
+    height: 30vh;
+    margin-bottom:-7px;
+    min-height:100px;
+    max-height:150px;
```

Comment:
Do these three do anything?

Reply:
In fact it seems to me that all of the rules in this selector don't do anything

Reply:
Correct - Removed All !